.gitignore

```diff
@@ -28,3 +28,6 @@ REVISION
 /doc/erd.*
 /app/assets/builds/*
 !/app/assets/builds/.keep
+node_modules/
+package.json
```

Comment:
Why would you add package.json and ignore it? (same for lock)

Reply:
I'm going to switch to global installing prettier and remove these. I just wanted to be able to install prettier within this directory and run it, but not have them check in.

These files weren't committed, just ignored so I could local install, but global is fine.